lib/TakeAway.rb

```diff
@@ -0,0 +1,10 @@
+class Takeaway
+
+	def process_order(order, payment)
+		if order.total_amount < payment
+			raise 'wrong payment' 
+		else
+			"thanks"
+		end
+	end
```

Comment:
Don't return a string for success - the brief does not ask for an interface. This method could simply raise an error or not raise an error.